apps/authentication-portal/src/main/webapp/totp.jsp

```diff
@@ -45,8 +45,20 @@
 
     if (Boolean.parseBoolean(request.getParameter(Constants.AUTH_FAILURE))) {
         authenticationFailed = "true";
-
-        if (request.getParameter(Constants.AUTH_FAILURE_MSG) != null) {
+        String errorCode = request.getParameter("errorCode");
+
+        if ( errorCode != null) {
+            if (errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE)) {
+                String lockedReason = request.getParameter("lockedReason");
+                if (lockedReason!=null) {
```

Comment:
Add a space between != `(lockedReason != null)`

Reply:
Addressed in 5492179bd28d858e7c17b98e645c5d9f340efc1a